SpecflowPages/Pages/Languages.cs

```diff
@@ -0,0 +1,42 @@
+﻿using OpenQA.Selenium;
+
+namespace Onboarding_Task_Mars_Specflow.SpecflowPages.Pages
+{
+    public class Languages
+    {
+        // XPath selectors for elements in the language section
+        public static By AddNewLanguageButton => By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/thead/tr/th[3]/div");
+        public static By AddLanguageTextBox => By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/div/div[1]/input");
+        public static By LanguageLevelDropDown => By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/div/div[2]/select");
+        public static By LanguageLevelOption => By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/div/div[2]/select/option[4]"); // Fluent level
+        public static By AddNewLanguageAndLevelButton => By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/div/div[3]/input[1]");
+
+        // XPath selectors for edit and delete
+        public static By EditLanguageIcon => By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody/tr[last()]/td[3]/span[1]");
+        public static By LanguageTextInput => By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody/tr/td/div/div[1]/input");
+        public static By LanguageLevelDropDownEdit => By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody/tr/td/div/div[2]/select");
+        public static By LanguageLevelOptionEdit => By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody/tr/td/div/div[2]/select/option[3]"); // Conversational level
```

Comment:
you should not make these variables static

Reply:
and they should not be marked public 
please read up on when to mark instance variables public vs private and same goes for static vs non static

Reply:
here is an also an example https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html
 of how you could layout the framework better